src/rpc/request.cpp

```diff
@@ -170,7 +170,7 @@ void DeleteAuthCookie()
             fs::remove(GetAuthCookieFile());
         }
     } catch (const fs::filesystem_error& e) {
-        LogPrintf("%s: Unable to remove random auth cookie file: %s\n", __func__, fsbridge::get_filesystem_error_message(e));
+        LogWarning("Unable to remove random auth cookie file %s: %s\n", fs::PathToString(e.path1()), e.code().message());
```

Comment:
non blocking nit, I think we can use `e.what()` in place of `e.code().message()`

Reply:
UTF-8 paths included in `e.what()` are not displayed correctly on Windows.

Reply:
> UTF-8 paths included in e.what() are not displayed correctly on Windows.

Maybe they will after #32380? In any case, including the filename doesn't seem necessary.